src/egl/drivers/dri2/egl_dri2.c

```diff
@@ -695,8 +695,6 @@ dri2_setup_screen(_EGLDisplay *disp)
       dri2_renderer_query_integer(dri2_dpy,
                                   __DRI2_RENDERER_HAS_CONTEXT_PRIORITY);
 
-   disp->Extensions.EXT_pixel_format_float = EGL_TRUE;
```

Comment:
does it work if you only remove this line?

Reply:
this would be nice because we could leave the plumbing in place and just not enable extension yet before float buffers are exposed